internal/controllers/nft_controller.go

```diff
@@ -115,10 +118,15 @@ func (nc *NFTController) GetNFTMetadata(c *fiber.Ctx) error {
 		name = description
 	}
 
+	imageURI := fmt.Sprintf("%s/v1/vehicle/%s/image", nc.Settings.DeploymentBaseURL, ti)
+	if !nc.Settings.IsProduction() && ud.IpfsImageCid.Valid {
+		imageURI = fmt.Sprintf("%s/%s", ipfs.Scheme, ud.IpfsImageCid.String)
```

Comment:
This would result in `ipfs:///CiDwhatvEr`, yeah? Maybe a function that goes `cid` → URL just to centralize this?